src/cmd/initialize_test.go

```diff
@@ -0,0 +1,33 @@
+// DEPRECATED_V1.0.0: do not check pkgConfig.InitOpts.RegistryInfo.NodePort, always overwrite it.
+package cmd
+
+import (
+	"testing"
+)
+
+func TestSetRegistryStorageClass(t *testing.T) {
+	// Test case 1: storageClass is set
+	pkgConfig.PkgOpts.SetVariables = map[string]string{"REGISTRY_STORAGE_CLASS": "test-storage-class"}
+	storageClassArg = ""
+	setRegistryStorageClass()
+	if storageClassArg != "test-storage-class" {
+		t.Errorf("Expected storage class to be set to 'test-storage-class', but got '%s'", storageClassArg)
```

Comment:
We should use `require.Equal()` to be more consistent with the rest of our test suite assertions. When `require.Equal()` fails it prints an "expected but got" message by default for you.